examples/hello_world/proto/BUILD

```diff
@@ -33,10 +35,22 @@ cc_proto_library(
     deps = [":hello_world_proto"],
 )
 
+java_proto_library(
+    name = "hello_world_java_proto",
+    deps = [":hello_world_proto"],
+)
+
 cc_grpc_library(
```

Comment:
As far as I can tell, there is no option for a 'lite' flavor or adding 'android' as a constraint, so probably not usable for an Android example.